src/annotation.ts

```diff
@@ -84,15 +83,15 @@ export class Annotation extends Modifier {
     let maxRightGlyphWidth = 0;
     for (let i = 0; i < annotations.length; ++i) {
       const annotation = annotations[i];
-      const textFormatter = TextFormatter.create(annotation.textFont);
+      const measure = Font.measureText(annotation.text, annotation.textFont!);
```

Comment:
perhaps refactor from `measure` to `measurement`?  `measure` has a very specific implication in music applications.